cmd/operator-sdk/generate/bundle/cmd.go

```diff
@@ -111,11 +111,13 @@ func (c *bundleCmd) addCommonFlagsTo(fs *pflag.FlagSet) {
 	fs.StringVar(&c.operatorName, "operator-name", "", "Name of the bundle's operator")
 	fs.StringVarP(&c.version, "version", "v", "", "Semantic version of the operator in the generated bundle. "+
 		"Only set if creating a new bundle or upgrading your operator")
-	fs.StringVar(&c.inputDir, "input-dir", "", "Directory to read an existing bundle format from")
-	fs.StringVar(&c.outputDir, "output-dir", "", "Directory to write the bundle format to")
-	fs.StringVar(&c.manifestRoot, "manifest-root", "", "Root directory for operator manifests, ex. Deployment and RBAC")
+	fs.StringVar(&c.inputDir, "input-dir", "", "Directory to read an existing bundle from. "+
+		"This directory is the parent of your bundle 'manifests' directory, and different from --manifest-root")
+	fs.StringVar(&c.outputDir, "output-dir", "", "Directory to write the bundle to")
+	fs.StringVar(&c.manifestRoot, "manifest-root", "", "Root directory for operator manifests, "+
+		"ex. 'deploy' or 'config'. This directory is different from that passed to --input-dir")
```

Comment:
I think it's still worth pointing out what manifests we're primarily looking for here i.e `Deployment, RBAC, Example CustomResources`.
That makes it more obvious that these are inputs manifests to building the CSV itself.

Also `manifest-root` is still pretty vague imo but let's circle back to this if we can't think of anything better.